homeassistant/components/hue/config_flow.py

```diff
@@ -143,6 +146,9 @@ def __init__(self):
         This flow is triggered by the SSDP component. It will check if the
         host is already configured and delegate to the import step if not.
         """
+        if discovery_info[ATTR_MANUFACTURERURL] == DECONZ_MANUFACTURERURL:
```

Comment:
We should not exclude Deconz urls, instead we should only include Hue urls.

Reply:
Done